src/main/resources/application.properties

```diff
@@ -0,0 +1,7 @@
+fixedRate.in.milliseconds=60000
+initialDelay.in.milliseconds=60000
+
+management.endpoints.web.exposure.include.=*
+
+jasypt.encryptor.bean=encryptorBean
+mpCryptoPassword=IUCDSPilot
```

Comment:
there is not much meaning of encrypting if we are gonna put the password in the properties file. For testing it might be acceptable but normally the password cannot be in the properties file.

one of options is to pass it as vm parameter.
eg. 
mvn-Djasypt.encryptor.password=secretkey spring-boot:run